openpype/hosts/maya/plugins/publish/collect_fbx_animation.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from maya import cmds  # noqa
+import pyblish.api
+
+
+class CollectFbxAnimation(pyblish.api.InstancePlugin):
+    """Collect Unreal Skeletal Mesh."""
+
+    order = pyblish.api.CollectorOrder + 0.2
+    label = "Collect Fbx Animation"
+    families = ["animation"]
+
+    def process(self, instance):
+        frame = cmds.currentTime(query=True)
+        instance.data["frameStart"] = frame
+        instance.data["frameEnd"] = frame
```

Comment:
I am not entirely sure about this - shouldn't this collect frame range and not the single frame? If there is some specific reason, I would put it in the comment.